tests/functional/skip_staging_table/source/data/in/tables/stagingasqdqwd.csv

```diff
@@ -0,0 +1,20 @@
+"1","connection.north-europe.azure.keboola.com","2","connection.north-europe.azure.keboola.com","2020-12-01 08:09:10","test 1"
+"2","connection.north-europe.azure.keboola.com","2","connection.north-europe.azure.keboola.com","2020-12-02 08:09:10","test 2"
+"3","connection.north-europe.azure.keboola.com","2","connection.north-europe.azure.keboola.com","2020-12-03 08:09:10","test 3"
+"4","connection.north-europe.azure.keboola.com","2","connection.north-europe.azure.keboola.com","2020-12-04 08:09:10","test 4"
+"5","connection.north-europe.azure.keboola.com","2","connection.north-europe.azure.keboola.com","2020-12-01 08:10:10","test 5"
+"1","connection.north-europe.azure.keboola.com","3","connection.north-europe.azure.keboola.com","2020-12-01 08:10:10","test 6"
```

Comment:
I don't understand how this is testing skipping staging tables

Reply:
@pivnicek v testu ve složce `/source/data/in/tables/` jsou tabulky které se před testováním nahrajou do snflk

a potom otestuju že se tabulka - v tomhle případě s názvem `stagingasqdqwd` nevyexportuje extractorem (přeskočí se)

Reply:
tabulky před testováním se nahrávají tady - https://github.com/keboola/ex-telemetry-data/blob/master/tests/functional/DatadirTest.php#L80

Reply:
ahaaaa takhle, diky moc, uz chapu 🍻